### PR TITLE
Lint staged: checks pre-commit

### PR DESCRIPTION
**Pull request by Crashillo**
### What does this PR do?
Includes a new git hooks, to check whether the staged files fulfill the rules, before do any commit

### CODE REVIEW

File: config/locales/gobierto_dashboards/views/es.yml [hunk 1]
```diff
@@ -13,6 +13,7 @@ es:
 
       Luego podrás configurarla."
     objective: Objetivo
+    ole: 1
```
Comment (ferblape) [inline config/locales/gobierto_dashboards/views/es.yml]: Careful

**Review by ferblape — COMMENTED** [review]
Will the linter affect to old code?

File: config/locales/gobierto_dashboards/views/es.yml (hunk 1 above)
Comment (Crashillo) [reply config/locales/gobierto_dashboards/views/es.yml]: It means: **O**bject **l**inted **e**xactly = 1 (true) :trollface:

**Comment by Crashillo** [comment]
No, that's the key. `lint-staged` seeks only in the staged files.
In order to check all files, you simply run: `npm run eslint` or `npm run stylelint`, but.... be pacient

**Comment by Crashillo** [comment]
CircleCI shows multiple failing tests, but apparently they're not related at all 🤷‍♂️

**Pull request merged**